### ISSUE TITLE
Single Token Cache breaks using multiple connections on the same machine

### ISSUE THREAD

**Issue by JefTek** [microsoftgraph/msgraph-sdk-powershell#553]
If you are signing into a Windows machine with a user, launching a PowerShell session and connect-mggraph to TenantA with UserA,  it works as expected.

You then open another PowerShell Window/Tab, and connect-Mggraph to TenantB with UserB,  if you then go back to PS window where you had TenantA connected it will require you to sign in again.   If you sign in, you will break the session for TenantB.

The issue appears to be the BIN3 cache file stored in the windows users .graph folder.    It will be cached from the last connection, and break any previous connections.   

This makes working with multiple tenants and connections very difficult since it all relies on the same cache file.   As an enhancement can there be a way to establish a per tenant and per user cache file so you can have multiple sessions active?

To resolve this, you need to delete the BIN3 file stored in the .graph folder else you get in an authentication prompt loop between cmdlet execution. 
[AB#8103](https://microsoftgraph.visualstudio.com/0985d294-5762-4bc2-a565-161ef349ca3e/_workitems/edit/8103)

**Comment by finsharp** [comment]
@JefTek `Connect-MgGraph` has a `-ContextScope` parameter which you can set to `Process` instead of `User`, allowing you to work with different tenants. This option doesn't write to the local cache. Would this be suitable to your scenario?

**Comment by JefTek** [comment]
> @JefTek `Connect-MgGraph` has a `-ContextScope` parameter which you can set to `Process` instead of `User`, allowing you to work with different tenants. This option doesn't write to the local cache. Would this be suitable to your scenario?

@finsharp  Yes,  I just tested this and I could run the sessions side by side on the same machine connected to 2 different tenants.  One was my regular, where I did not specify context, and the 2nd I specified context as process and there were no collisions.   Thank you!     Can we add a section on token cache and context to this doc?  https://docs.microsoft.com/en-us/graph/powershell/get-started
